### ISSUE TITLE
XR005 seems to not be working for my provider

### ISSUE THREAD

**Issue by remyleone** [bflad/tfproviderlint#342]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I got a provider that should trigger warnings for XR005 https://github.com/scaleway/terraform-provider-scaleway/ because there are many resources that do not have any description fill in
But the linter does not give any warnings.

### New or Affected Checks(s)

* XR005

### Flagged Provider Code

```go
// flagged code
```

### Passing Provider Code

```go
// passing code
```

### References

* Links if you have any!
